### ISSUE TITLE
Make Exercise Boxes Consistent

### ISSUE THREAD

**Issue by jrleeman** [Unidata/python-training#38]
We've got a mix of non green background exercise boxes (which do work in jupyter lab now) and no background boxes. Which shall we choose to standardize on?

**Comment by dopplershift** [comment]
Capturing our conversation: Since the green boxes work, let's use them since they IMO look nicer.

**Comment by jrleeman** [comment]
Relevant HTML:

```html
<div class="alert alert-success">
    <b>EXERCISE</b>:
     <ul>
         <li></li>
         <li></li>
         <li></li>
    </ul>
</div>



<div class="alert alert-info">
    <b>SOLUTION</b>
</div>
```

**Comment by dopplershift** [comment]
Closed by #69.
